src/logics/ArithLogic.cc

```diff
@@ -193,6 +202,15 @@ bool ArithLogic::isLinearTerm(PTRef tr) const {
     return false;
 }
 
+bool ArithLogic::isNonlin(PTRef tr) const {
```

Comment:
This method should not be needed anymore, no?

Reply:
Removed